pages/linux/rtcwake.md

```diff
@@ -0,0 +1,36 @@
+# rtcwake
+
+> Enter a system sleep state until specified wakeup time relative to your bios clock.
+> Commands need to be run as superuser since shutdowns are involved.
+
+- Check your hardware clock info and exit:
+
+`rtcwake -m show -v`
+
+- Show if an alarm is set:
+
+`rtcwake -m show`
+
+- Disable a previous set alarm:
+
+`rtc -m disable`
+
+- Suspend to ram and wakeup after 10 seconds:
+
+`sudo rtcwake -m mem -s {{10}}`
+
+- Suspend to disk (higher power saving) and wakeup 10 minutes later:
```

Comment:
The description mentions 10 minutes, but example has 15 minutes. Please choose one.

Also, we should be consistent in the language. The description before has "after 10 seconds", but this one has "10 minutes later". We should choose one and be consistent.

Reply:
Yes, I thought that adding more examples was better for a higher understanding of how the thing works.. so s for seconds, m for minutes and so on.. Or maybe Is better a longer description?

Reply:
Or maybe a standard time like 1h and the thing written in 3 different ways in 3 different examples like: 3600s, 60m and 1h!

Reply:
We already have 15m and 1h, so I think it is clear enough that different units can be used.